assessment.js

```diff
@@ -1,8 +1,7 @@
 //We'll learn about require later in the course
 var ask = require('./ask.js');
+var i = 1;
 
-var answer = 'not empty';
-
-while (answer !== '' && answer !== 'SeCrEt') {
-  answer = ask("Guess my secret? ");
+while (i >= 1 && i <= 10) {
+  console.log(i); i++;
```

Comment:
Not quite what was asked, though admittedly the question could have been worded more clearly. We were looking more for something along the lines of 
```javascript
var i = -1;
while (i !== 7) {
  answer = ask("Guess a number between 1 and 10.");
}
```